### DISCUSSION TITLE
New jinn-codemirror modes

### DISCUSSION THREAD

**Discussion by timathom** [Jinntec/Fore#210] [Ideas]
I'd like to add a new mode to the `jinn-codemirror` component for syntax highlighting of MARC records. The mode has [already been implemented](https://gbv.github.io/lipstick/codemirror/), so I can probably use that and pass in the data with a template expression, but it would be nice to be able to easily extend the `jinn-codemirror` component.

**Comment by JoernT** [comment]
This should be moved to the https://github.com/JinnElements/jinn-codemirror repo. I added a feature-request backlinking here. 

https://github.com/JinnElements/jinn-codemirror/issues/7 Feel free to comment.
